turbo-txpool/src/lib.rs

```diff
@@ -87,3 +228,138 @@ impl server::Txpool for TxPool {
         Box::pin(self.get_transactions(request))
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    fn inner() -> Inner {
+        let cfg = Config::builder().max_txs(10).build();
+        Inner::with_config(cfg)
+    }
+
+    fn tx(gas_price: u64) -> Tx {
+        Tx {
+            gas_price: gas_price.into(),
+            gas: Default::default(),
+            hash: H256::from_low_u64_be(gas_price),
```

Comment:
Any particular rationale for this? Just to have something to differentiate by?

Reply:
Exactly, yeah.